aws-rds-cfn-common/pom.xml

```diff
@@ -72,7 +72,7 @@
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
             <version>4.3.1</version>
-            <scope>test</scope>
```

Comment:
Do we need mockito in runtime?

Reply:
This is the downside of keeping Test utils in cfn-common: any test utility dependency bubbles up into a non-test classpath. An alternative to that would be to split cfn-common-core and cfn-common-test.